src/assets/_project/_blocks/utils/qg-load-google-api.js

```diff
@@ -2,9 +2,27 @@
  * When using functionality related to google libraries, this fuction comes handy to check if libraries already exists and then execute custom function
  */
 /* globals qg */
+import keys from '../qg-google-keys';
+
 (function (qg, $) {
   'use strict';
-  // lazy load a script
+  var findFranchiseName = function () {
+    var path = window.location.pathname.replace(/\/$/, '');
+    var pathArr = path.split('/').filter(function (e) {
```

Comment:
to lower case before split.

Reply:
@william-dutton-dsiti 
I have added    **return pathArr[0].toLowerCase()**; in the latest commit
Instead of before split I added to the value we need because if we lowercase things we don't need that would increase the time.

Reply:
you may also need a test to ensure that there is a item in the array. (defensive coding)

Reply:
I have added a condition to check and then run the loop, empty array will not break the code because undefined is not equal to string but the side effect is that it will run the loop when there is no need

let franchise = findFranchiseName();
  if (franchise) {